### PR TITLE
Update VS/SDK support table

### LINKED ISSUE #32350: Need to add .NET 7 GA support
The table needs to be updated .NET 7 GA support to 17.4

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: d5366311-80dd-f979-ae35-3b9fba19578d
* Version Independent ID: ff588613-155b-81a8-94b9-855df04d24d9
* Content: [Breaking change: Version requirements for .NET 7 SDK - .NET](https://learn.microsoft.com/en-us/dotnet/core/compatibility/sdk/7.0/vs-msbuild-version)
* Content Source: [docs/core/compatibility/sdk/7.0/vs-msbuild-version.md](https://github.com/dotnet/docs/blob/main/docs/core/compatibility/sdk/7.0/vs-msbuild-version.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @gewarren
* Microsoft Alias: **gewarren**

### PR DESCRIPTION
**Pull request by timheuer**
Fixes #32350 by adding GA/17.4 support

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by timheuer** [comment]
@marcpopMSFT

**Review by marcpopMSFT — APPROVED** [review]

**Comment by marcpopMSFT** [comment]
fwiw, I don't see any changes to https://github.com/dotnet/docs/blob/main/docs/core/porting/versioning-sdk-msbuild-vs.md needed at the moment.

**Review by gewarren — COMMENTED** [review]
Line 16 should be updated too (and perhaps the ms.date metadata).

File: docs/core/compatibility/sdk/7.0/vs-msbuild-version.md [hunk 1]
```diff
@@ -13,13 +13,14 @@ Certain .NET SDK preview and release candidate versions require newer versions o
...
 
-The following table shows the minimum version of Visual Studio and MSBuild you'll need to use .NET 7.0.100 SDK Preview 3, Preview 7, and RC 2.
+The following table shows the minimum version of Visual Studio and MSBuild you'll need to use .NET 7.0.100 SDK Preview 3, Preview 7, RC 2, and GA.
 
 | NET 7.0.100 SDK version | Minimum Visual Studio and MSBuild version |
 |-------------------------|-------------------------------------------|
 | Preview 3               | 17.0<sup>1</sup>                          |
 | Preview 7               | 17.2                                      |
 | RC 2                    | 17.3                                      |
+| GA                      | 17.4                          |
```
Comment (IEvangelist) [inline docs/core/compatibility/sdk/7.0/vs-msbuild-version.md:23]:
```suggestion
| RC 2                    | 17.3                                      |
| GA                      | 17.4                                      |
```

**Review by IEvangelist — APPROVED** [review]

**Pull request merged**